.travis.yml

```diff
@@ -174,28 +177,30 @@ jobs:
     if: type IN (pull_request)
     env:
     - NODE_ENV=production
-    - BUNDLE_ANALYSIS=1
     - PORT=12345
-    - VERSION=$TRAVIS_PULL_REQUEST_BRANCH
-    - SENTRY_ENVIRONMENT=pr
+    - VERSION=$TRAVIS_BRANCH
+    - SENTRY_ENVIRONMENT=staging
```

Comment:
If this is the PR branch builds, why change these?
    - VERSION=$TRAVIS_PULL_REQUEST_BRANCH
    - SENTRY_ENVIRONMENT=pr

Reply:
ah, copy paste error - I should have left these in 🤦‍♂

Reply:
Yep. We switched a while back to be 'more production settings like' but now with a proper private blockchain I think it's a good time to switch back to be like staging.

Reply:
Ok, please push those new lines if you are making changes to here or let me know if I should fix them 👍

Reply:
Just doing a last sanity check, then I'll push those lines and merge.

Reply:
Perfect, thanks 🎉